R/rmapi/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "rmapi"
```

Comment:
Do you want to call this `rMAPI`?

Reply:
I do t have preference, the repo is all small case but the readme is not lol.